2.1.4 -> 2.2.0rc2 - wget issue

I get some kind of a strange error...

It comes from a `wget` which tries to reach https://raw.githubusercontent.com/angular/angular.js/704123a/src/ngLocale/angular-locale_fr-ch%0D.js, but it should rather try  https://raw.githubusercontent.com/angular/angular.js/704123a/src/ngLocale/angular-locale_fr-ch.js

Any idea from where the `%0D` comes ??

```
wget -O sitn/static-ngeo/build/angular-locale_fr.js https://raw.githubusercontent.com/angular/angular.js/`grep ^angular.js= node_modules/ngeo/github_versions | cut --delimiter = --fields 2`/src/ngLocale/angular-locale_`(grep fr language_mapping || echo fr) | cut -d = -f 2`.js
--2017-06-16 14:11:42--  https://raw.githubusercontent.com/angular/angular.js/704123a/src/ngLocale/angular-locale_fr-ch%0D.js
Resolving raw.githubusercontent.com... 151.101.36.133
Connecting to raw.githubusercontent.com|151.101.36.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-06-16 14:11:42 ERROR 404: Not Found.

make[1]: *** [CONST_Makefile:821: sitn/static-ngeo/build/angular-locale_fr.js] Error 8
make[1]: Leaving directory '/cygdrive/c/git_projects/sitn_c2cgeoportal'

[01;32m================================================================[0m
[01;31mThe command '['make', '-f', 'michael.mk', 'build']' returns the error code 2.[0m
[01;32mFix it and run it again:[0m
[01;32mmake -f michael.mk upgrade4[0m
```

--- a/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl
+++ b/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl
@@ -71,12 +71,12 @@ endif
 export INSTANCE_ID
 
 PIP_CMD ?= $(VENV_BIN)/pip
-PIP_VERSION ?= $(shell grep ^pip== CONST_optional-requirements.txt --colour=never)
-SETUPTOOLS_VERSION ?= $(shell grep ^setuptools== CONST_optional-requirements.txt --colour=never)
-TILECLOUD_VERSION ?= $(shell grep ^tilecloud== CONST_optional-requirements.txt --colour=never)
-TILECLOUDCHAIN_VERSION ?= $(shell grep ^tilecloud-chain== CONST_optional-requirements.txt --colour=never)
-JSTOOLS_VERSION ?= $(shell grep ^JSTools== CONST_optional-requirements.txt --colour=never)
-C2CCSSMIN_VERSION ?= $(shell grep ^c2c.cssmin== CONST_optional-requirements.txt --colour=never)
+PIP_VERSION ?= $(shell grep ^pip== CONST_optional-requirements.txt --colour=never | tr --delete '\r\n')
+SETUPTOOLS_VERSION ?= $(shell grep ^setuptools== CONST_optional-requirements.txt --colour=never | tr --delete '\r\n')
+TILECLOUD_VERSION ?= $(shell grep ^tilecloud== CONST_optional-requirements.txt --colour=never | tr --delete '\r\n')
+TILECLOUDCHAIN_VERSION ?= $(shell grep ^tilecloud-chain== CONST_optional-requirements.txt --colour=never | tr --delete '\r\n')
+JSTOOLS_VERSION ?= $(shell grep ^JSTools== CONST_optional-requirements.txt --colour=never | tr --delete '\r\n')
+C2CCSSMIN_VERSION ?= $(shell grep ^c2c.cssmin== CONST_optional-requirements.txt --colour=never | tr --delete '\r\n')
 
 GIT_MODULES_FOLDER ?= .git/modules/
 
@@ -818,7 +818,7 @@ $(APP_OUTPUT_DIR)/%.json: $(PACKAGE)/locale/%/LC_MESSAGES/$(PACKAGE)-$(L10N_CLIE
 
 $(APP_OUTPUT_DIR)/angular-locale_%.js: node_modules/ngeo/github_versions language_mapping
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/angular/angular.js/`grep ^angular.js= $< | cut --delimiter = --fields 2`/src/ngLocale/angular-locale_`(grep $* language_mapping || echo $*) | cut -d = -f 2`.js
+	wget -O $@ https://raw.githubusercontent.com/angular/angular.js/`grep ^angular.js= $< | cut --delimiter = --fields 2 | tr --delete '\r\n'`/src/ngLocale/angular-locale_`(grep $* language_mapping || echo $*) | cut --delimiter = --fields 2 | tr --delete '\r\n'`.js
 
 $(APP_OUTPUT_DIR)/%.js: .build/%.js .build/node_modules.timestamp $(NGEO_LIBS_JS_FILES) $(NGEO_PROJECT_LIBS_JS_FILES)
 	$(PRERULE_CMD)
@@ -959,7 +959,7 @@ $(APP_OUTPUT_DIR)/templatecache.js: .build/gmftemplatecache.js
 .build/externs/jquery-1.9.js: node_modules/ngeo/github_versions
 	$(PRERULE_CMD)
 	mkdir -p $(dir $@)
-	curl --max-redirs 0 --location --output $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/jquery-1.9.js
+	curl --max-redirs 0 --location --output $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2  | tr --delete '\r\n'`/contrib/externs/jquery-1.9.js
 
 package.json:
 	$(PRERULE_CMD)

